src/vdaf/prg.rs

```diff
@@ -124,8 +124,10 @@ pub struct PrgAes128(Cmac<Aes128>);
 impl Prg<16> for PrgAes128 {
     type SeedStream = SeedStreamAes128;
 
-    fn init(seed_bytes: &[u8; 16]) -> Self {
-        Self(Cmac::new_from_slice(seed_bytes).unwrap())
+    fn init(seed_bytes: &[u8; 16], custom: &[u8]) -> Self {
+        let mut mac = Cmac::new_from_slice(seed_bytes).unwrap();
+        mac.update(custom);
```

Comment:
Should we length-delimit `custom` here? I suppose that PrgAes128 is no longer constrained by the draft. I think all of our customization strings in practice are eight bytes, but this could be a sharp edge if we change that before this updated PrgAes128 goes out of use.

Reply:
Yeah I think that's probably a good idea. I was trying to maintain backwards compatibility with the PrgAes128 unit tests in the reference implementation, but I think safety is more important.

Reply:
https://github.com/cfrg/draft-irtf-cfrg-vdaf/pull/157